### DISCUSSION TITLE
How do I make a DateTime snippet with the format I need?

### DISCUSSION THREAD

**Discussion by jhon65496** [tamlok/vnote#2488] [Q&A]
How do I make a DateTime snippet with the format I need?

Now the `DateTime` snippet inserts the text `2024-04-14_19:23:50`.

How do I create a `DateTime2` snippet so that it inserts the text `20240414192350`?

![image](https://github.com/vnotex/vnote/assets/128128859/6542e58d-a00d-45ba-ab90-5253b58cf64a)

**Comment by tamlok** [comment]
`%yyyy%%MM%%dd%%HH%%mm%%ss%`
